docs/cmdline-opts/connect-timeout.d

```diff
@@ -3,6 +3,8 @@ Arg: <seconds>
 Help: Maximum time allowed for connection
 See-also: max-time
 Category: connection
+Example: --connect-timeout 20 $URL
+Example: --connect-timeout 3.14 $URL
```

Comment:
While not the fault of this PR, I wonder if we should change `Arg:` to be `<fractional seconds>` or something which indicates the resolution.

Reply:
Sounds fair, and the same for `--max-time` then...